lib/remap-lang/src/lib.rs

```diff
@@ -169,6 +169,35 @@ mod tests {
                 "#,
                 Ok(Some(true.into())),
             ),
+            (r#"if false { 1 }"#, Ok(None)),
+            (r#"if true { 1 }"#, Ok(Some(1.into()))),
+            (r#"if false { 1 } else { 2 }"#, Ok(Some(2.into()))),
+            (r#"if false { 1 } else if false { 2 }"#, Ok(None)),
+            (r#"if false { 1 } else if true { 2 }"#, Ok(Some(2.into()))),
+            (
+                r#"if false { 1 } else if false { 2 } else { 3 }"#,
+                Ok(Some(3.into())),
+            ),
+            (
+                r#"if false { 1 } else if true { 2 } else { 3 }"#,
+                Ok(Some(2.into())),
+            ),
+            (
+                r#"if false { 1 } else if false { 2 } else if false { 3 }"#,
+                Ok(None),
+            ),
+            (
+                r#"if false { 1 } else if false { 2 } else if true { 3 }"#,
+                Ok(Some(3.into())),
+            ),
+            (
+                r#"if false { 1 } else if true { 2 } else if false { 3 } else { 4 }"#,
+                Ok(Some(2.into())),
+            ),
+            (
+                r#"if false { 1 } else if false { 2 } else if false { 3 } else { 4 }"#,
+                Ok(Some(4.into())),
+            ),
```

Comment:
Tiny point. I think it would be worth adding a test for:

```
if false { 1 } else { if false { 2 } else { 3 } } else { 5 }
```

If we ever decide to make surrounding blocks with '{}' optional, this test might help remind us how bad an idea that might be!